municipal_finance/resources.py

```diff
@@ -0,0 +1,60 @@
+from import_export import resources
+
+from .models import (
+    AmountTypeV2,
+    CflowItemsV2,
+    GovernmentFunctionsV2,
+    IncexpItemsV2,
+    BsheetItemsV1,
+    BsheetItemsV2,
+    BsheetFactsV1,
+    BsheetFactsV2,
+)
+
+
+class AmountTypeV2Resource(resources.ModelResource):
+    class Meta:
+        model = AmountTypeV2
+        import_id_fields = ['code']
+
+
+class CashflowItemsV2Resource(resources.ModelResource):
+    class Meta:
+        model = CflowItemsV2
+        import_id_fields = ['code']
+
+
+class GovernmentFunctionsV2Resource(resources.ModelResource):
+    class Meta:
+        model = GovernmentFunctionsV2
+        import_id_fields = ['code']
+
+
+class IncexpItemsV2Resource(resources.ModelResource):
+    class Meta:
+        model = IncexpItemsV2
+        import_id_fields = ['code']
+
+
+class BsheetItemsV1Resource(resources.ModelResource):
+    class Meta:
+        model = BsheetItemsV1
+        import_id_fields = ['code']
+
+
+class FinancialPositionItemsV2Resource(resources.ModelResource):
+    class Meta:
+        model = BsheetItemsV2
+        import_id_fields = ['code']
+
+
+class BsheetFactsV2Resource(resources.ModelResource):
+    class Meta:
+        model = BsheetFactsV2
+        import_id_fields = ['id']
+
+
+class BsheetFactsV1Resource(resources.ModelResource):
+    class Meta:
+        model = BsheetFactsV1
+        import_id_fields = ['id']
```

Comment:
is this just for dev? we wouldn't have data with ids in normal operations - this makes me a bit nervous that it's a dev-specific thing but not clearly marked as such, and in normal production code.

Reply:
These are only used in tests, and will probably not be used for uploading real data (too slow and memory hungry). I have however made a commit that indicated the correct fields to lookup on.